develop-env/base/Dockerfile

```diff
@@ -65,30 +65,69 @@ RUN source /opt/lsst/software/stack/loadLSST.bash && setup lsst_distrib &&\
+    conda install -y ipython jupyter jupyterlab
 
 RUN source /opt/lsst/software/stack/loadLSST.bash && setup lsst_distrib &&\
     conda install -y jinja2 ephem gitpython
 # jsonschema=3.0.1
 
 RUN source /opt/lsst/software/stack/loadLSST.bash && setup lsst_distrib &&\
-    conda update -y -n base -c defaults conda && \
-    conda update -y pyqt fontconfig sqlalchemy && \
-    conda install -y -c anaconda-platform asynctest && \
-    conda install -y -c conda-forge pytest-asyncio && \
-    conda install -y -c conda-forge kafkacat moto boto3 && \
-    conda install -y conda-build &&\
-    conda install -y -c conda-forge pytest-flake8 && \
-    conda clean -ay && \
-    pip install "documenteer[pipelines]==0.4.2" && \
-    pip install "kafkit[aiohttp]" && \
-    pip install aiokafka && \
-    pip install confluent_kafka && \
-    pip install asynccmd && \
-    pip install aiounittest && \
-    pip install ipdb && \
+    conda config --add channels conda-forge
+
+#RUN source /opt/lsst/software/stack/loadLSST.bash && setup lsst_distrib &&\
+#    conda install -c anaconda setuptools
+
+#RUN source /opt/lsst/software/stack/loadLSST.bash && setup lsst_distrib &&\
+#    conda update -y -n base -c defaults conda
+
+RUN source /opt/lsst/software/stack/loadLSST.bash && setup lsst_distrib &&\
+    conda update -y pyqt fontconfig sqlalchemy
+
+RUN source /opt/lsst/software/stack/loadLSST.bash && setup lsst_distrib &&\
+    conda install -y -c anaconda-platform asynctest
+
+RUN source /opt/lsst/software/stack/loadLSST.bash && setup lsst_distrib &&\
+    conda install -y -c conda-forge pytest-asyncio
+
+RUN source /opt/lsst/software/stack/loadLSST.bash && setup lsst_distrib &&\
+    conda install -y -c conda-forge kafkacat moto boto3
+
+RUN source /opt/lsst/software/stack/loadLSST.bash && setup lsst_distrib &&\
+    conda install -y conda-build
+
+RUN source /opt/lsst/software/stack/loadLSST.bash && setup lsst_distrib &&\
+    conda install -y -c conda-forge pytest-flake8
+
+#RUN source /opt/lsst/software/stack/loadLSST.bash && setup lsst_distrib &&\
+#    conda install -c conda-forge pytest-subtests
+
+RUN source /opt/lsst/software/stack/loadLSST.bash && setup lsst_distrib &&\
+    pip install "documenteer[pipelines]==0.4.2"
+
+RUN source /opt/lsst/software/stack/loadLSST.bash && setup lsst_distrib &&\
+    pip install "kafkit[aiohttp]"
+
+RUN source /opt/lsst/software/stack/loadLSST.bash && setup lsst_distrib &&\
+    pip install aiokafka
+
+RUN source /opt/lsst/software/stack/loadLSST.bash && setup lsst_distrib &&\
+    pip install confluent_kafka
+
+RUN source /opt/lsst/software/stack/loadLSST.bash && setup lsst_distrib &&\
+    pip install asynccmd
+
+RUN source /opt/lsst/software/stack/loadLSST.bash && setup lsst_distrib &&\
+    pip install aiounittest
+
+RUN source /opt/lsst/software/stack/loadLSST.bash && setup lsst_distrib &&\
+    pip install ipdb
+
+RUN source /opt/lsst/software/stack/loadLSST.bash && setup lsst_distrib &&\
     pip install wget
```

Comment:
Why do you split all of these RUN commands? Because there is a limit to the number of layers in docker image.

Reply:
Two reasons, I was trying to keep the layers as small as possible to help downloading the images and, in case one of the installation fails, I know which one it is. I had some of them failing for some reason.